website/layouts/partials/under-construction-tmp.html

```diff
@@ -0,0 +1,16 @@
+<div class="alert alert-warning" role="alert">
+	<div class="row">
+		<div class="col-1">
+			<i class="fas fa-hammer"></i>
+		</div>
+		<div class="col-11">
+			<p class="m-0">
+				This site may contain outdated or incomplete information.
+			</p>
+			<p class="m-0">
+				We aim to have all content updated before 
```

Comment:
I feel since this is opensource work, we can make this more generic to say in the coming weeks, instead of fixed deadline. We still also in my opinion think that we should add the website issue link to point folks who want to help, so that they know where to get started or show interest. Even if two folks organically join and show interest that's better than nothing :)

Reply:
"Coming weeks" requires a point-in-time to give it meaning. I'll add a link to the GitHub issue per your suggestion... that should be sufficient to remove any time references.